lib/platform/gitlab/index.ts

```diff
@@ -936,20 +936,27 @@ export async function addAssignees(
 ): Promise<void> {
   logger.debug(`Adding assignees '${assignees.join(', ')}' to #${iid}`);
   try {
-    let assigneeId = await getUserID(assignees[0]);
-    let url = `projects/${config.repository}/merge_requests/${iid}?assignee_id=${assigneeId}`;
-    await gitlabApi.putJson(url);
-    try {
-      if (assignees.length > 1) {
-        url = `projects/${config.repository}/merge_requests/${iid}?assignee_ids[]=${assigneeId}`;
-        for (let i = 1; i < assignees.length; i += 1) {
-          assigneeId = await getUserID(assignees[i]);
-          url += `&assignee_ids[]=${assigneeId}`;
+    if (assignees.length === 1) {
+      logger.debug('Setting single assignee');
+      const assigneeId = await getUserID(assignees[0]);
+      const url = `projects/${config.repository}/merge_requests/${iid}?assignee_id=${assigneeId}`;
+      await gitlabApi.putJson(url);
+    } else {
+      try {
+        logger.debug('Try using gitlab premium API');
```

Comment:
```suggestion
        logger.debug('Try using GitLab premium API');
```

GitLab is a proper noun so should be capitalized. 😉 

But why would a user need to "Try using GitLab premium API"? The message is not really clear about this. Do we need to expand the debug message to explain _why_?

Reply:
Hi thanks for the question, i cant say for sure if Gitlab supports `assignee_ids` on all versions now but in https://docs.gitlab.com/ee/development/documentation/restful_api_styleguide.html#method-description its described as Premium (or Enterprise) only thus i dont want to fail setting any assignee and leave the user without any hint.

Reply:
There's a info-icon next to `assignee_ids` which says it is:

> Available in GitLab Premium self-managed, GitLab Premium SaaS, and higher tiers.

So GitLab only allows `assignee_ids` on some tiers, not all of them.